python/ruff/__main__.py

```diff
@@ -1,7 +1,8 @@
 import os
 import sys
 import sysconfig
+from pathlib import Path
 
 if __name__ == "__main__":
-    ruff = os.path.join(sysconfig.get_path("scripts"), "ruff")
+    ruff = Path(sysconfig.get_path("scripts")) / "ruff"
```

Comment:
Is this change related to prettier / markdownlint?

Reply:
No, but it was failing the Ruff hook. So I fixed it.